### DISCUSSION TITLE
github dependebot "create security update" doesn't work

### DISCUSSION THREAD

**Discussion by exx8** [community/community#5360] [Code Security]
I've notified correctly that I have a vuln. dependencies, in the security tab. But whenever I press "create security update" nothing happens.
![image](https://user-images.githubusercontent.com/8540180/131345178-3ac62fda-1c05-474e-bd02-2c7eed51e838.png)
It get stuck for a while in this mode:
![image](https://user-images.githubusercontent.com/8540180/131345222-1640cc3a-79cb-412b-b6d3-087869d8b4d5.png)

then returns to normal, as nothing has happened.

**Comment by jhutchings1** [comment]
Do you mind opening a support issue so we can investigate this particular failure? https://support.github.com/
